db/seeds.d/020-roles_list.rb

```diff
@@ -4,10 +4,13 @@ class RolesList
   class << self
     def seeded_roles
       {
-        Role::MANAGER => { :permissions => base_manage_permissions + view_permissions + manage_organizations_permissions,
+        Role::MANAGER => { :permissions => base_manage_permissions + view_permissions + manage_organizations_permissions + settings_permissions,
                            :description => 'Role granting all available permissions. With this role, user is able to do everything that admin can except for changing settings.' },
         Role::ORG_ADMIN => { :permissions => base_manage_permissions + view_permissions,
                              :description => 'Role granting all permissions except for managing organizations. It can be used to delegate administration of specific organization to a user. In order to create such role, clone this role and assign desired organizations' },
+        Role::CANNED_ADMIN => { :permissions => (settings_permissions + manage_organizations_permissions + canned_admin_extra_permissions + escalate_roles_permission),
+                                :description => 'Role granting permissions for managing organizations, users and settings.' },
```

Comment:
Does not block testing but the description should mention other added resources and potentially a fact, this is a powerful role that can potentially get access to all resources.